clients/spark/core/src/main/scala/io/treeverse/clients/GarbageCollector.scala

```diff
@@ -249,24 +297,29 @@ object GarbageCollector {
   }
 
   def main(args: Array[String]) {
-    val spark = SparkSession.builder().getOrCreate()
     if (args.length != 2) {
       Console.err.println(
         "Usage: ... <repo_name> <region>"
       )
       System.exit(1)
     }
 
+    val spark = SparkSession.builder().appName("GarbageCollector").getOrCreate()
+
     val repo = args(0)
     val region = args(1)
     val previousRunID =
       "" //args(2) // TODO(Guys): get previous runID from arguments or from storage
     val hc = spark.sparkContext.hadoopConfiguration
+
+    val hcValues = spark.sparkContext.broadcast(getHadoopConfigurationValues(hc, "fs."))
```

Comment:
it's probably obvious for Spark experts but I think that we can benefit from documenting why we use broadcast here.

Reply:
No idea about Spark experts, so did it.